lib/px/utils/pxtelemetry.lua

```diff
@@ -0,0 +1,32 @@
+local px_common_utils = require('px.utils.pxcommonutils')
+local px_constants = require('px.utils.pxconstants')
+local hmac = require "resty.nettle.hmac"
+
+local M = {}
+
+function M.telemetry_check_header(px_config, px_client, px_headers, px_logger)
```

Comment:
why do you need to pass `px_client` and `px_config` as params? you can just require them

Reply:
It requires calling load() method with px_config, seemed like double work to me. wdyt?

Reply:
if that's the case, then I guess not